6.4_remember_remember.py

```diff
@@ -0,0 +1,32 @@
+from PIL import Image
+import os.path
+
+MY_PATH = "resources//code.png"
+
+
+def decryption(path):
+    """
+    Gets a path of an image and decodes a code hidden in pixels
+    :param path: path of image
+    :return: code 
+    """
+    if os.path.exists(path):
+        try:
+            im = Image.open(path)
+        except IOError:
+            return "the path is not an image file"
+        pix = im.load()
+        width, hight = im.size
+        code = ""
+        for i in range(width):
```

Comment:
Try to avoid i and j as well if there is a better name available.

Reply:
I changed i to row and j to column